.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+version: 2.1
+executors:
+  infrastructure_container:
+    docker:
+      - image: aeternity/infrastructure
+    working_directory: /src
+
+commands:
+  integration_tests:
+    description: "Integration Tests"
+    steps:
+      - run:
+          name: Terraform init
+          command: |
+            cd test && terraform init
+      - run:
+          name: Terraform lint and validate
+          command: |
+            cd test && terraform fmt -check=true -diff=true && terraform validate
+      - run:
+          name: Integration environment setup
+          command: |
+            cd test && terraform apply -parallelism=20 --auto-approve
+      - run:
+          name: Integration environment health check
+          command: |
+            ENVID=${TF_VAR_envid:?} ./test/health-check.sh
+      - run:
+          name: Integration environment cleanup
+          command: |
+            cd test && terraform destroy -parallelism=20 --auto-approve
+          when: always
+
+  setup_secrets:
+    steps:
+      - run:
+          name: Setup environment secrets
+          command: echo "source /infrastructure/import-secrets.sh" >> $BASH_ENV
```

Comment:
I see this repo terraform-aws-aenode-deploy used in several points in aeternity/infrastructure repo. Which repo depends on which? It seems chicken-and-egg to me.